cypress/integration/test-helper.js

```diff
@@ -0,0 +1,5 @@
+export const testRequestResponse = (uri, responseCode) => {
```

Comment:
This helper function should be called something more specific, i.e. `testResponseCode`. Also the first argument should be the `path`.

Finally, we use `camelCasing` as a naming convention, not `kebab-casing`. so just rename to `testHelpers`.